src/pre-init-utils.ts

```diff
@@ -1,5 +1,16 @@
+import { IPixelConfiguration } from './cookieSyncManager';
+import { MPForwarder } from './forwarders.interfaces';
+import { IntegrationDelays } from './mp-instance';
 import { isEmpty, isFunction } from './utils';
 
+export interface IPreInit {
+    readyQueue: Function[];
```

Comment:
[based on the usage below](https://github.com/mParticle/mparticle-web-sdk/pull/968/files#diff-da27b40568cc9938393028855aed1db2e8fc2b9e7bb803504e270241477d178cR19-R21), this can be an array of functions, or an array of arrays.  I believe the second half is when someone tries to call `mParticle.something` before `something` fully loads (we stub these in the snippet).  might take some more digging into, but this is what i recall.